pwiz_tools/Skyline/FileUI/PublishDocumentDlg.cs

```diff
@@ -99,7 +106,7 @@ private void PublishDocumentDlg_Load(object sender, EventArgs e)
             foreach (var serverFolder in listServerFolders)
             {
                 var server = serverFolder.Key;
-                var treeNode = new TreeNode(server.URI.ToString()) { Tag = new FolderInformation(server, false) };
+                var treeNode = new TreeNode(server.GetKey()) { Tag = new FolderInformation(server, false) };

```

Comment:
Use a class or struct instead of a KeyValuePair (or Tuple) for lists shared between functions.